audiofile: cannot build with lcov option

- [x] Confirmed this is a problem with `brew install`ing one, specific Homebrew/homebrew-core formula (not cask or tap) and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/caskroom/homebrew-cask/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ ] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?
----
To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
```
$ brew install -s audiofile --with-lcov
==> Downloading https://audiofile.68k.org/audiofile-0.3.6.tar.gz
######################################################################## 100.0%
==> Downloading https://mirrors.ocf.berkeley.edu/debian/pool/main/a/audiofile/au
######################################################################## 100.0%
==> Downloading https://mirrors.ocf.berkeley.edu/debian/pool/main/a/audiofile/au
Already downloaded: /Users/aoyamatakuo/Library/Caches/Homebrew/audiofile--patch-0620675a52bdb40b775980cc1820e308df329348bb847f9a4a8361b3799fa241.tar.xz
==> Patching
==> Applying patches/03_CVE-2015-7747.patch
patching file libaudiofile/modules/ModuleState.cpp
patching file test/Makefile.am
patching file test/sixteen-stereo-to-eight-mono.c
==> Applying patches/04_clamp-index-values-to-fix-index-overflow-in-IMA.cpp.patc
patching file libaudiofile/modules/IMA.cpp
==> Applying patches/05_Always-check-the-number-of-coefficients.patch
patching file libaudiofile/WAVE.cpp
==> Applying patches/06_Check-for-multiplication-overflow-in-MSADPCM-decodeSam.p
patching file libaudiofile/modules/BlockCodec.cpp
patching file libaudiofile/modules/MSADPCM.cpp
==> Applying patches/07_Check-for-multiplication-overflow-in-sfconvert.patch
patching file sfcommands/sfconvert.c
==> Applying patches/08_Fix-signature-of-multiplyCheckOverflow.-It-returns-a-b.p
patching file libaudiofile/modules/MSADPCM.cpp
patching file sfcommands/sfconvert.c
==> Applying patches/09_Actually-fail-when-error-occurs-in-parseFormat.patch
patching file libaudiofile/WAVE.cpp
==> Applying patches/10_Check-for-division-by-zero-in-BlockCodec-runPull.patch
patching file libaudiofile/modules/BlockCodec.cpp
==> ./configure --prefix=/usr/local/Cellar/audiofile/0.3.6_1 --enable-coverage
==> make
Last 15 lines from /Users/s172262/Library/Logs/Homebrew/audiofile/02.make:
/bin/sh ../libtool --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall  -fprofile-arcs -ftest-coverage  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o units.lo units.cpp
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fprofile-arcs -ftest-coverage -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c units.cpp  -fno-common -DPIC -o .libs/units.o
/bin/sh ../libtool --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..    -DNDEBUG -Wall  -fprofile-arcs -ftest-coverage  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c -o util.lo util.cpp
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fprofile-arcs -ftest-coverage -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c util.cpp  -fno-common -DPIC -o .libs/util.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fprofile-arcs -ftest-coverage -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c units.cpp -o units.o >/dev/null 2>&1
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -DNDEBUG -Wall -fprofile-arcs -ftest-coverage -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -c util.cpp -o util.o >/dev/null 2>&1
/bin/sh ../libtool --tag=CXX   --mode=link clang++ -DNDEBUG -Wall  -fprofile-arcs -ftest-coverage  -fvisibility=hidden -fno-rtti -fno-exceptions -fvisibility-inlines-hidden -g -O2 -version-info 1:0:0 -no-undefined -export-symbols ./audiofile.exports  -o libaudiofile.la -rpath /usr/local/Cellar/audiofile/0.3.6_1/lib AIFF.lo AVR.lo AudioFormat.lo Buffer.lo CAF.lo FLACFile.lo File.lo FileHandle.lo IFF.lo IRCAM.lo Instrument.lo Loop.lo Marker.lo Miscellaneous.lo NIST.lo NeXT.lo PacketTable.lo Raw.lo SampleVision.lo Setup.lo Track.lo UUID.lo VOC.lo WAVE.lo aes.lo af_vfs.lo aupv.lo compression.lo data.lo debug.lo error.lo extended.lo format.lo g711.lo openclose.lo pcm.lo query.lo units.lo util.lo modules/libmodules.la alac/libalac.la -lgcov  -lm 
libtool: link: sed 's,^,_,' < ./audiofile.exports > .libs/libaudiofile-symbols.expsym
libtool: link: clang++ -dynamiclib  -o .libs/libaudiofile.1.dylib  .libs/AIFF.o .libs/AVR.o .libs/AudioFormat.o .libs/Buffer.o .libs/CAF.o .libs/FLACFile.o .libs/File.o .libs/FileHandle.o .libs/IFF.o .libs/IRCAM.o .libs/Instrument.o .libs/Loop.o .libs/Marker.o .libs/Miscellaneous.o .libs/NIST.o .libs/NeXT.o .libs/PacketTable.o .libs/Raw.o .libs/SampleVision.o .libs/Setup.o .libs/Track.o .libs/UUID.o .libs/VOC.o .libs/WAVE.o .libs/aes.o .libs/af_vfs.o .libs/aupv.o .libs/compression.o .libs/data.o .libs/debug.o .libs/error.o .libs/extended.o .libs/format.o .libs/g711.o .libs/openclose.o .libs/pcm.o .libs/query.o .libs/units.o .libs/util.o   -Wl,-force_load,modules/.libs/libmodules.a -Wl,-force_load,alac/.libs/libalac.a  -lgcov -lm  -fprofile-arcs -O2   -install_name  /usr/local/Cellar/audiofile/0.3.6_1/lib/libaudiofile.1.dylib -compatibility_version 2 -current_version 2.0 -Wl,-single_module -Wl,-exported_symbols_list,.libs/libaudiofile-symbols.expsym
ld: library not found for -lgcov
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [libaudiofile.la] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
`libgcov.a` is included `gcc`. But `gcc` formula don't build `libgcov.dylib`. 
Is there another way to build `libgcov.dylib`? Or give up and remove this option?

--- a/Formula/audiofile.rb
+++ b/Formula/audiofile.rb
@@ -33,12 +33,9 @@ class Audiofile < Formula
     depends_on "libtool" => :build
   end
 
-  option "with-lcov", "Enable Code Coverage support using lcov"
   option "with-test", "Run the test suite during install (~30sec)"
 
   deprecated_option "with-check" => "with-test"
-
-  depends_on "lcov" => :optional
 
   # These have all been reported upstream but beside
   # 03_CVE-2015-7747 not yet merged or fixed.
@@ -75,7 +72,6 @@ class Audiofile < Formula
 
     configure = build.head? ? "./autogen.sh" : "./configure"
     args = ["--disable-dependency-tracking", "--prefix=#{prefix}"]
-    args << "--enable-coverage" if build.with? "lcov"
     system configure, *args
     system "make"
     system "make", "check" if build.with? "test"

